packages/babel-plugin-ember-test-metadata/__tests__/utils-test.js

```diff
@@ -32,111 +29,120 @@ describe('getNodeProperty', () => {
 });
 
 describe('getNormalizedFilePath', () => {
-  const fileOpts = {
-    embroiderBuildPath: {
-      root: '',
-      filename: path.join(
-        'private',
-        'var',
-        'folders',
-        'abcdefg1234',
-        'T',
-        'embroider',
-        '098765',
-        'tests',
-        'acceptance',
-        'my-test.js'
-      ),
-    },
-    embroiderBuildPathTwoEmbroiderPathSegments: {
-      root: '',
-      filename: path.join(
-        'private',
-        'var',
-        'folders',
-        'embroider',
-        'abcdefg1234',
-        'T',
-        'embroider',
-        '098765',
-        'tests',
-        'acceptance',
-        'my-test.js'
-      ),
-    },
-    normalFilePath: {
-      root: '',
-      filename: path.join('this', 'is', 'not-an-embroider', 'path'),
-    },
-  };
-  const projectConfiguration = {
-    pkg: {
-      name: 'test-app',
-      'ember-addon': {},
-    },
-  };
-
-  it('returns stripped file path as expected', () => {
-    expect(getNormalizedFilePath(fileOpts.embroiderBuildPath, projectConfiguration)).toBe(
-      path.join('tests', 'acceptance', 'my-test.js')
-    );
-    expect(
-      getNormalizedFilePath(
-        fileOpts.embroiderBuildPathTwoEmbroiderPathSegments,
-        projectConfiguration
-      )
-    ).toBe(path.join('tests', 'acceptance', 'my-test.js'));
-  });
+  let appRoot;
 
-  it('returns unmodified file path when path does not include "embroider" as a segment', () => {
-    expect(getNormalizedFilePath(fileOpts.normalFilePath, projectConfiguration)).toBe(
-      path.join('this', 'is', 'not-an-embroider', 'path')
-    );
-  });
-});
+  describe("given file path from a classic build", () => {
+    beforeEach(() => {
+      appRoot = "/Users/tester/workspace/personal/test-bed/classic";
+    });
 
-describe('_getParsedClassicFilepath', () => {
-  it('returns file path from classic build correctly', () => {
-    expect(
-      _getParsedClassicFilepath(['test-app', 'tests', 'acceptance', 'my-test.js'], {
+    it("returns the normalized filepath", () => {
+      const expectedPath = "tests/acceptance/foo-test.js";
+      const preFormattedPath = "/Users/tester/workspace/personal/test-bed/classic/classic/tests/acceptance/foo-test.js";
+      const opts = { filename: preFormattedPath, root: "/Users/tester/workspace/personal/test-bed/classic" };
+      const config = {
         pkg: {
-          name: 'test-app',
-          'ember-addon': {},
+          name: "classic"
         },
-      })
-    ).toBe(path.join('tests', 'acceptance', 'my-test.js'));
-    expect(
-      _getParsedClassicFilepath(
-        ['@parent-repo-name', 'test-app', 'tests', 'acceptance', 'my-test.js'],
-        {
-          pkg: {
-            name: `@parent-repo-name${path.sep}test-app`,
-            'ember-addon': {},
-          },
-        }
-      )
-    ).toBe(path.join('tests', 'acceptance', 'my-test.js'));
+      }
+
+      const formattedPath = getNormalizedFilePath(opts, config);
```

Comment:
Can we name this `normalizedFilePath`, since you're calling a function called `getNormalizedFilePath`, which is intended to return a `normalizedFilePath`.